.circleci/config.yml

```diff
@@ -10,7 +10,11 @@ jobs:
                     python3 -m venv venv
                     . venv/bin/activate
                     pip install requests
-                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ test/optimus/pr
+                    python test/run_portability_test.py --test-directories test/smartseq2_single_sample/pr/ \
+                                                                           test/smartseq2_single_sample/fldm/
```

Comment:
After you update your branch from master, can you also add this as a separate test below that uses `./test/trigger_test.sh`? I can help you do this.